### PR TITLE
SUPERSET-8: Update text in docs copyright footer

### PR DESCRIPTION
**Pull request by andrewmusselman**
### CATEGORY

- [ * ] Documentation

### SUMMARY
Docs footer has extra text and the year is out-dated:
© Copyright Copyright © 2018 The Apache Software Foundation, Licensed under the Apache License, Version 2.0..

### TEST PLAN
Verify footer says "Copyright © 2019 The Apache Software Foundation, Licensed under the Apache License, Version 2.0." with the correct year.

### CODE REVIEW

**Comment by andrewmusselman** [comment]
I don't know why there are commits on here from two branches; thought I'd branched off master for both.

In any case they're both very small.

**Comment by mistercrunch** [comment]
Please use descriptive titles to your PRs
https://chris.beams.io/posts/git-commit/

**Comment by mistercrunch** [comment]
I think we want that copyright in the docs don't we?

File: docs/conf.py [hunk 1]
```diff
@@ -194,7 +194,7 @@
 html_show_sphinx = False
 
 # If true, "(C) Copyright ..." is shown in the HTML footer. Default is True.
-html_show_copyright = True
+html_show_copyright = False
```
Comment (andrewmusselman) [inline docs/conf.py]: This line, when set to "True", was wrapping another copyright symbol and period around the text in line 68, so I believe setting it to "False" removes that.

**Comment by andrewmusselman** [comment]
See description in jira: https://issues.apache.org/jira/browse/SUPERSET-8
"© Copyright Copyright © 2018 The Apache Software Foundation, Licensed under the Apache License, Version 2.0.."
Setting that show copyright flag to False I believe will remove that wrapped copyright symbol and word, plus the extra period at the end.

**Pull request merged**